### PR TITLE
Update CsTeamsComplianceRecordingPolicy

### PR DESCRIPTION
**Pull request by abnan**
Updating documentation for CsTeamsComplianceRecordingPolicy to add new policy bit

### CODE REVIEW

File: skype/skype-ps/skype/New-CsTeamsComplianceRecordingPolicy.md [hunk 1]
```diff
@@ -152,6 +153,22 @@ Accept wildcard characters: False
...
+
+Required: False
+Position: Named
+Default value: False
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -RecordReroutedCalls
+Setting this attribute to true enables compliance recording for calls which have been re-routed from a compliance recording enabled user. Supported call scenarios include forward, transfer, delegation, call groups, simultaneous ring.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/New-CsTeamsComplianceRecordingPolicy.md:169]:
```suggestion
Setting this attribute to true enables compliance recording for calls that have been re-routed from a compliance recording-enabled user. Supported call scenarios include forward, transfer, delegation, call groups, and simultaneous ring.

```

File: skype/skype-ps/skype/Set-CsTeamsComplianceRecordingPolicy.md [hunk 2]
```diff
@@ -212,6 +212,22 @@ Accept wildcard characters: False
...
+
+Required: False
+Position: Named
+Default value: False
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -RecordReroutedCalls
+Setting this attribute to true enables compliance recording for calls which have been re-routed from a compliance recording enabled user. Supported call scenarios include forward, transfer, delegation, call groups, simultaneous ring.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsTeamsComplianceRecordingPolicy.md:228]:
```suggestion
Setting this attribute to true enables compliance recording for calls that have been re-routed from a compliance recording-enabled user. Supported call scenarios include forward, transfer, delegation, call groups, and simultaneous ring.

```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @abnan, kindly commit the suggested changes. Many thanks.

File: skype/skype-ps/skype/New-CsTeamsComplianceRecordingPolicy.md [hunk 3]
```diff
@@ -152,6 +153,23 @@ Accept wildcard characters: False
...
+Default value: False
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -RecordReroutedCalls
+Setting this attribute to true enables compliance recording for calls that have been re-routed from a compliance recording-enabled user. Supported call scenarios include forward, transfer, delegation, call groups, and simultaneous ring.
+
+
+
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/New-CsTeamsComplianceRecordingPolicy.md:172]:
```suggestion
```

File: skype/skype-ps/skype/Set-CsTeamsComplianceRecordingPolicy.md [hunk 4]
```diff
@@ -212,6 +212,23 @@ Accept wildcard characters: False
...
+Default value: False
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -RecordReroutedCalls
+Setting this attribute to true enables compliance recording for calls that have been re-routed from a compliance recording-enabled user. Supported call scenarios include forward, transfer, delegation, call groups, and simultaneous ring.
+
+
+
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsTeamsComplianceRecordingPolicy.md:231]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Apologies @abnan, please also commit these suggestions to remove the extra blank lines. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by dariomws — APPROVED** [review]

**Comment by yogkumgit** [comment]
@get-itips @tseward

**Review by get-itips — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**